examples/sampling/pyg/node_classification.py

```diff
@@ -0,0 +1,259 @@
+    #   device: The device (CPU/GPU) to run the training on.
+    #####################################################################
+
+    model.train()  # Set the model to training mode
+    total_loss = 0  # Accumulator for the total loss
+    total_correct = 0  # Accumulator for the total number of correct predictions
+    total_samples = 0  # Accumulator for the total number of samples processed
+    num_batches = 0  # Counter for the number of mini-batches processed
+
+    for minibatch in dataloader:
+        node_features = minibatch.node_features["feat"]
+        labels = minibatch.labels
+        optimizer.zero_grad()
+        out = model(minibatch.blocks, node_features, device)
+        loss = criterion(out, labels)
+        total_loss += loss.item()
+        total_correct += compute_accuracy(out, labels) * labels.size(0)
+        total_samples += labels.size(0)
+        loss.backward()
+        optimizer.step()
+        num_batches += 1
+    avg_loss = total_loss / num_batches
+    avg_accuracy = total_correct / total_samples
+    return avg_loss, avg_accuracy
+
+
+@torch.no_grad()
+def evaluate(model, dataloader, device, num_classes):
+    model.eval()
+    y_hats = []
+    ys = []
+    for minibatch in dataloader:
+        node_features = minibatch.node_features["feat"]
+        labels = minibatch.labels
+        out = model(minibatch.blocks, node_features, device)
+        y_hats.append(out)
+        ys.append(labels)
+
+    return MF.accuracy(
+        torch.cat(y_hats),
+        torch.cat(ys),
+        task="multiclass",
+        num_classes=num_classes,
+    )
+
+
+@torch.no_grad()
+def layerwise_infer(graph, feature, test_set, model, num_classes, device):
+    model.eval()
+    dataloader = create_dataloader(
+        dataset_set=test_set,
+        graph=graph,
+        feature=feature,
+        device=device,
+        shuffle=False,
+    )
+    all_predictions = []
+    all_labels = []
+
+    for minibatch in dataloader:
+        node_features = minibatch.node_features["feat"]
+        labels = minibatch.labels
+        blocks = minibatch.blocks
+        predictions = model(blocks, node_features, device)
+        all_predictions.append(predictions)
+        all_labels.append(labels)
+
+    all_predictions = torch.cat(all_predictions, dim=0)
+    all_labels = torch.cat(all_labels, dim=0)
+
+    return MF.accuracy(
+        all_predictions, all_labels, task="multiclass", num_classes=num_classes
+    )
+
+
+def main():
+    dataset = gb.BuiltinDataset("ogbn-arxiv").load()
+    graph = dataset.graph
+    feature = dataset.feature
+    train_set = dataset.tasks[0].train_set
+    valid_set = dataset.tasks[0].validation_set
+    test_set = dataset.tasks[0].test_set
+    num_classes = dataset.tasks[0].metadata["num_classes"]
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    train_dataloader = create_dataloader(
+        train_set, graph, feature, device, True
```

Comment:
```suggestion
        train_set, graph, feature, device, is_train=True
```
So that the user knows what that parameter is for. It is more readable.